tests/org.jboss.tools.cdi.bot.test/src/org/jboss/tools/cdi/bot/test/beansxml/discovery/cdi11/BeanDiscoveryInImplicitArchivesTest.java

```diff
@@ -60,21 +48,6 @@
 	
 	@Test
 	public void testValidationOfBeanDiscoveryInImplicitArchives() {
-		prepareBeanXml("annotated", false);
-		ProjectExplorer pe = new ProjectExplorer();
-		pe.open();
-		pe.getProject(PROJECT_NAME).getProjectItem(CDIConstants.JAVA_RESOURCES, CDIConstants.SRC, "test", "CdiBean1.java").open();
-		TextEditor ed = new TextEditor("CdiBean2.java");
-		try{
-			new WaitUntil(new EditorHasValidationMarkers(ed));
-		} catch (WaitTimeoutExpiredException ex){
-			fail("There is supposed to be warning present");
-		}
-		List<Marker> markers = ed.getMarkers();
-		assertEquals(1, markers.size());
-		Marker validation = markers.get(0);
-		assertEquals(validation.getLineNumber(), 7);
-		assertTrue(validation.getText().contains("No bean is eligible for injection"));
+		super.validationOfBeanDiscoveryInImplicitArchives();
```

Comment:
Add CDI11 pattern to the class name. Also, In new CDI20 test classes you have specified CDIVersion, add it to the CDI 11 classes as well, of course, value will be for 1.2 or 1.1.

Reply:
CDIVersion specification add to the CDI 11 classes.

Reply:
Is it necessary to rename **super.validationOfBeanDiscoveryInImplicitArchives();** and **super.validationOfBeanDiscoveryInExplicitArchives();** by adding **CDI11** and **CDI20**? Both CDI11 and CDI20 test classes are using the same test methods collected in template class (BeanDiscoveryInArchivesTemplate). The will be duplicity - either there will be 2 same functions **validationOfBeanDiscoveryInExplicitArchives();** and 2 same function for **validationOfBeanDiscoveryInImplicitArchives();** in the template class or there will be duplicity in the classes, that extends the template class.

Reply:
CDI11 pattern added to the class name.